### PR TITLE
Add note about system config in php

### PR DESCRIPTION
**Pull request by Cruiser13**
Add new recommended way to read system config in controllers to docs

### CODE REVIEW

File: doc/Development_Documentation/18_Tools_and_Features/25_System_Settings.md [hunk 1]
```diff
@@ -123,3 +123,29 @@ Settings for outbound HTTP connectivity of Pimcore - needed e.g. for Pimcore Upd
...
+class DefaultController extends FrontendController
+{
+ 
+    /**
+     * @param Request $request
+     */
+    public function defaultAction(Request $request)
+    {
+    
+        $pimcoreconfig = $this->getParameter('pimcore.config');
```
Comment (dvesh3) [inline doc/Development_Documentation/18_Tools_and_Features/25_System_Settings.md:145]: type-hinting config class should be the recommended way. please see https://github.com/pimcore/pimcore/blob/ea7c3906e85a71af0138b67ee8b242b3a29599f2/bundles/AdminBundle/Controller/Admin/DataObject/DataObjectHelperController.php#L313

File: doc/Development_Documentation/18_Tools_and_Features/25_System_Settings.md [hunk 2]
```diff
@@ -123,3 +123,29 @@ Settings for outbound HTTP connectivity of Pimcore - needed e.g. for Pimcore Upd
 Possibility for configuring different newsletter delivery settings from the default e-mail settings.
  
  
+## Access system config in PHP Controller
+Using \Pimcore\Config::getSystemConfig(); is deprecated. Access the system config using the default symfony way in your controllers:
```
Comment (jdreesen) [inline doc/Development_Documentation/18_Tools_and_Features/25_System_Settings.md:127]:
```suggestion
Using `\Pimcore\Config::getSystemConfig()` is deprecated. Access the system config using the default Symfony way in your controllers:

```

File: doc/Development_Documentation/18_Tools_and_Features/25_System_Settings.md (hunk 1 above)
Comment (brusch) [reply doc/Development_Documentation/18_Tools_and_Features/25_System_Settings.md:146]: Both ways are fine, but using the config service is probably a bit more handy 😉

Comment (brusch) [reply doc/Development_Documentation/18_Tools_and_Features/25_System_Settings.md:146]: I guess we should just include both ways in the docs, then it's up to the dev 😊

Comment (brusch) [reply doc/Development_Documentation/18_Tools_and_Features/25_System_Settings.md:146]: @Cruiser13 could you please update your PR and include the alternative way described by @dvesh3 
Thanks a lot!

File: doc/Development_Documentation/18_Tools_and_Features/25_System_Settings.md [hunk 3]
```diff
@@ -123,3 +123,32 @@ Settings for outbound HTTP connectivity of Pimcore - needed e.g. for Pimcore Upd
...
+
+use Pimcore\Controller\FrontendController;
+use Symfony\Component\HttpFoundation\Request;
+use Pimcore\Config;
+
+class DefaultController extends FrontendController
+{
+ 
+    /**
+     * @param Request $request
```
Comment (jdreesen) [inline doc/Development_Documentation/18_Tools_and_Features/25_System_Settings.md:142]: I'd say the DocBlock should either be removed or `@param Config $config` should be added here.

Comment (jdreesen) [reply doc/Development_Documentation/18_Tools_and_Features/25_System_Settings.md:142]: Oh, it's already merged...

Comment (brusch) [reply doc/Development_Documentation/18_Tools_and_Features/25_System_Settings.md:142]: 👍 thanks, will push that straight to master

File: doc/Development_Documentation/18_Tools_and_Features/25_System_Settings.md (hunk 1 above)
Comment (Cruiser13) [reply doc/Development_Documentation/18_Tools_and_Features/25_System_Settings.md]: Sorry, I didn't see your message. Thanks for taking care of the docs!

**Pull request merged**